kernelci.org/content/en/contacts.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Contact us"
+date: 2024-09-12
+description: "How to contact us"
+weight: 1
+---
+
+> KernelCI community contacts are available on the [KernelCI website](https://kernelci.org/community-contact//).
```

Comment:
nitpick: Only needs a single / on the end of the URL.

Reply:
fixed